packages/fuel-indexer/src/executor.rs

```diff
@@ -74,18 +75,16 @@ impl ExecutorSource {
 pub fn run_executor<T: 'static + Executor + Send + Sync>(
     fuel_node_addr: &str,
     mut executor: T,
-    start_block: Option<u64>,
+    start_block: &u64,
     kill_switch: Arc<AtomicBool>,
     stop_idle_indexers: bool,
 ) -> impl Future<Output = ()> {
-    let start_block_value = start_block.unwrap_or(1);
```

Comment:
Removing this will cause a panic if a number is not used for the corresponding field in the manifest due to the `.unwrap()` call in `get_start_block()`.